zebrad/src/commands/start.rs

```diff
@@ -100,24 +92,33 @@ impl StartCmd {
         let inbound = ServiceBuilder::new()
             .load_shed()
             .buffer(20)
-            .service(Inbound::new(
-                setup_rx,
-                state.clone(),
-                chain_verifier.clone(),
-            ));
+            .service(Inbound::new(setup_rx));
 
         let (peer_set, address_book) =
             zebra_network::init(config.network.clone(), inbound, latest_chain_tip.clone()).await;
 
+        info!("initializing verifiers");
+        let (chain_verifier, tx_verifier, mut groth16_download_handle) =
+            zebra_consensus::chain::init(
+                config.consensus.clone(),
+                config.network.network,
+                state.clone(),
+            )
+            .await;
```

Comment:
The Inbound refactor lets us move this initialisation after the state and network startup.